frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/coordsProperty/coordsProperty.helpers.component.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { rgbaToHex } from '@/v4/helpers/colors';
+import { get, isArray, isObject } from 'lodash';
+import { theme } from '@/v5/ui/themes/theme';
+import { ITemplate } from '@/v5/store/tickets/tickets.types';
+import { contrastColor } from 'contrast-color';
+import { AdditionalProperties } from '../../../tickets.constants';
+
+const DEFAULT_COLOR = theme.palette.primary.main;
+
+const findByName = (array, name) => array.find(({ name: n }) => n === name);
+const rgbArrayToHex = (obj) => rgbaToHex(obj?.toString());
+
+const PROPERTIES = 'properties';
+export const DEFAULT_PIN = `${PROPERTIES}.${AdditionalProperties.PIN}`;
+
+type IGetPinSchema = {
+	name: string;
+	template: ITemplate;
+};
+const getPinSchema = ({ name, template }: IGetPinSchema) => {
```

Comment:
could you pass two parameters instead of an object of type `IGetPinSchema` and add the type definition for the return type?